### ISSUE TITLE
`apt-get update` fails in python runtimes due to jessie missing Release file

### ISSUE THREAD

**Issue by nickodell** [Azure/azure-functions-docker#874]
# Error

```
Sending build context to Docker daemon  3.689MB
Step 1/3 : FROM mcr.microsoft.com/azure-functions/python:4-python3.10
 ---> f29700f10b0e
Step 2/3 : RUN apt-get update
 ---> Running in 2d5be5eb8c82
Ign:1 http://security.debian.org/debian-security jessie/updates InRelease
Hit:2 http://deb.debian.org/debian bullseye InRelease
Err:3 http://security.debian.org/debian-security jessie/updates Release
  404  Not Found [IP: 146.75.106.132 80]
Get:4 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Get:5 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
Get:6 https://packages.microsoft.com/debian/9/prod stretch InRelease [1917 B]
Get:7 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [236 kB]
Get:8 https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages [137 kB]
Reading package lists...
E: The repository 'http://security.debian.org/debian-security jessie/updates Release' no longer has a Release file.
The command '/bin/sh -c apt-get update' returned a non-zero code: 100
```

# Expected behavior

I expect to be able to use `apt-get update` to get the latest version of dependencies.

# Example code

Example Dockerfile which triggers this:

```
FROM mcr.microsoft.com/azure-functions/python:4-python3.10
RUN apt-get update
```

# Workaround

Within `/etc/apt/sources.list`, one of the sources is

```
deb http://security.debian.org/debian-security jessie/updates main
```

One workaround is to remove the line, like so:

```
FROM mcr.microsoft.com/azure-functions/python:4-python3.10

RUN sed -i '/jessie/d' /etc/apt/sources.list \
 && apt-get update
```

This allows `apt-get update` to succeed.

# Root cause

As previously mentioned, the file `/etc/apt/sources.list` contains a line for jessie.

It seems that Debian has removed the Release file for jessie, which is understandable given that security updates for jessie stopped more than two years ago.

The odd part is - given that these images are based on either buster or bullseye, why is it attempting to add security updates for jessie, which is two releases earlier? The answer seems to be this comment:

https://github.com/Azure/azure-functions-docker/blob/d5afd1dc6df70e43ad0025064ce79af8d039672f/host/4/bullseye/amd64/python/python310/python310.Dockerfile#L70-L71

# Wrapping up

1. Is there any way you could remove or comment this line from sources.list in published images, after you've installed the software you need?
2. Distributing a copy of `libss1.0.0` from a distribution which no longer receives security updates seems not great? Is there any way you could use something more recent, from a distribution which does get security updates?

**Comment by andre-qumulo** [comment]
Thank you, we just found this today when all of our pipelines failed to build docker

**Comment by miqm** [comment]
Additionally to the original problem, line above https://github.com/Azure/azure-functions-docker/blob/d5afd1dc6df70e43ad0025064ce79af8d039672f/host/4/bullseye/amd64/python/python310/python310.Dockerfile#L69 also seems that can become a problem soon...

**Comment by SamuelYvon** [comment]
This is not great practice, but the package list are not empty on the base image. I was able to install what I needed (not much, so YMMV) without the `apt update`.

**Comment by GBBBAS** [comment]
Got the same error - switching to the nightly build worked `mcr.microsoft.com/azure-functions/python:4-nightly-python3.10`

**Comment by solita-kalleka** [comment]
How quickly does Microsoft usually react to these kinds of problems? Even though that workaround probably works, the existence of that jessie line is concerning, as it comes with the lack of security updates.

Hopefully someone from Microsoft responds to this thread. We can't currently build our functions app because of this, and we would like to know if it's necessary to make a new ticket about this.

**Comment by BerGod** [comment]
I have the same issues, for now i am running apt update -m ignoring the missing file and upgrading the rest, but i dont think is the best solution. Hope someone will answer soon.

**Comment by nickodell** [comment]
>How quickly does Microsoft usually react to these kinds of problems? Even though that workaround probably works, the existence of that jessie line is concerning, as it comes with the lack of security updates.

Looking at closed issues on this tracker of similar number of comments, it generally seems to take 1-2 weeks for a response. At that point we'll have a timeline for a fix. If you need it before then I would suggest finding a workaround.

**Comment by ColeHuntley83** [comment]
seeing the same for 3.0-python3.8 image.. Hopefully within the next 2 week there is a better approach then 

`RUN sed -i '/jessie/d' /etc/apt/sources.list`
and then apt-get update.

**Comment by charlie-normand** [comment]
We have the same issue, solved with the sed approach, but quite a drawback - can @microsoft @ahmelsayed @pragnagopa please get this thread some attention.  Thanks

**Comment by thivagar-m** [comment]
Ran into the same problem today while trying to build my project using docker file

**Comment by thivagar-m** [comment]
Faced the same issue today for python:4-python3.9 image.
The workaround was able to solve the problem currently. Hope to hear from microsoft soon

**Comment by pragnagopa** [comment]
public images listed in https://github.com/Azure/azure-functions-docker#readme have been updated. Closing this addressed.

**Comment by pragnagopa** [comment]
PR https://github.com/Azure/azure-functions-docker/pull/877 addressed the issue

**Comment by akrzysztofek** [comment]
Problem isn't solved by #877 for Buster - only Bullseye code path has been addressed there.
